README.md

```diff
@@ -64,23 +64,32 @@ This project can be run in a [Docker](https://www.docker.com/) container.
 You need a local database instance listening on port 8080. Here is how to set that up:
 
 ``` sh
-docker run --name mysql -e MYSQL_ROOT_PASSWORD=password -p 3306:3306 -d mysql
-export LEDGER_DB_URI=mysql://root:password@localhost/fivebells
-npm run migrate
+docker run --name db -e POSTGRES_PASSWORD=password -p 5432:5432 -d postgres
+export LEDGER_DB_URI=postgres://postgres:password@localhost
+docker run -it --rm --link db:postgres postgres psql -h postgres -U postgres
+docker run -it --rm --link db:postgres -v `pwd`:/src postgres psql -h postgres -U postgres
+	postgres=# \i /src/src/sql/pg/1-2.sql
```

Comment:
Total nitpick, but `/src/src` is a bit confusing, so why not mount it under `/var/app`, `/opt/five-bells-ledger` or something like that.